app/qml/components/MMCodeScanner.qml

```diff
@@ -0,0 +1,134 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtMultimedia
+import lc 1.0 as InputClass
+
+Drawer {
```

Comment:
Is there any reason to make this a drawer? It looks more like a full-screen page to me

Reply:
The drawer is the best item for it, I think. The Drawer for fullscreen is an item of a small-sized editor. So we don't need to push QR scanner item to any StackView